owlkettle/widgets.nim

```diff
@@ -128,17 +128,28 @@ renderable BaseWindow of BaseWidget:
 
 renderable Window of BaseWindow:
   title: string
+  fullscreened: bool ## Whether or not the window is fullscreened; setting this property is equivalent to calling `gtk_window_fullscreen` and `gtk_window_unfullscreen` for values of `true` and `false`, respectively. Either operation is asynchronous, which means you will need to connect to the `::notify` signal in order to know whether the operation was successful (it could fail if, for instance, a window manager refuses to respond to the signal).
```

Comment:
The docstring here seems to be copied from the GTK documentation. Since GTK itself it LGPL (and the docs are probably too, not sure), we cannot include it in owlkettle. The provided information also does not apply to owlkettle, since it is impossible to connect to the `::notify` signal in owlkettle. I would just remove the comment entirely.

Reply:
Ah, I didn't realize that. I'll remove it.